server/src/test/java/io/crate/integrationtests/RestSQLActionIntegrationTest.java

```diff
@@ -43,8 +43,8 @@ public void testWithoutBody() throws IOException {
         assertEquals(400, response.getStatusLine().getStatusCode());
         String bodyAsString = EntityUtils.toString(response.getEntity());
         assertThat(bodyAsString, startsWith("{\"error\":{\"message\":\"" +
-                                            "SQLActionException[SQLParseException: Missing request body]\"," +
-                                            "\"code\":4000},\"error_trace\":\"SQLActionException:"
```

Comment:
`SqlActionException` would disappear from the error message and the specific exception would be shown here.

Reply:
I think that's a good thing :+1: